Fail to compile with `default-features = false` on macOS

```rust
 error[E0433]: failed to resolve: use of undeclared crate or module `std`
 --> /Users/runner/.cargo/registry/src/github.com-1ecc6299db9ec823/nanorand-0.5.2/src/entropy/darwin.rs:1:5
  |
1 | use std::ffi::c_void;
  |     ^^^ use of undeclared crate or module `std`

error[E0433]: failed to resolve: use of undeclared crate or module `std`
  --> /Users/runner/.cargo/registry/src/github.com-1ecc6299db9ec823/nanorand-0.5.2/src/entropy/darwin.rs:10:30
   |
10 |     unsafe { SecRandomCopyBytes(std::ptr::null(), out.len(), out.as_mut_ptr()) == 0 }
   |                                 ^^^ use of undeclared crate or module `std`

error[E0412]: cannot find type `c_void` in this scope
 --> /Users/runner/.cargo/registry/src/github.com-1ecc6299db9ec823/nanorand-0.5.2/src/entropy/darwin.rs:5:36
  |
5 |     fn SecRandomCopyBytes(rnd: *const c_void, count: usize, bytes: *mut u8) -> u32;
  |                                       ^^^^^^ not found in this scope
  |
help: consider importing this enum
  |
1 | use core::ffi::c_void;
  |
```

Would it be better to add macOS ci build target ?

--- a/.github/workflows/tests.yml
+++ b/.github/workflows/tests.yml
@@ -21,7 +21,7 @@ jobs:
         uses: actions-rs/cargo@v1
         with:
           command: check
-          args: --no-default-features
+          args: --no-default-features --manifest-path=nanorand/Cargo.toml
       - name: Run 'cross test'
         uses: actions-rs/cargo@v1
         with:
@@ -61,7 +61,7 @@ jobs:
           uses: actions-rs/cargo@v1
           with:
             command: check
-            args: --no-default-features
+            args: --no-default-features --manifest-path=nanorand/Cargo.toml
         - name: Run 'cargo test'
           uses: actions-rs/cargo@v1
           with:
@@ -87,7 +87,7 @@ jobs:
           uses: actions-rs/cargo@v1
           with:
             command: check
-            args: --no-default-features
+            args: --no-default-features --manifest-path=nanorand/Cargo.toml
         - name: Run 'cargo test'
           uses: actions-rs/cargo@v1
           with:

--- a/.github/workflows/tests.yml
+++ b/.github/workflows/tests.yml
@@ -12,6 +12,16 @@ jobs:
           override: true
           profile: minimal
           toolchain: stable
+      - name: Run 'cargo check'
+        uses: actions-rs/cargo@v1
+        with:
+          command: check
+          args: --all-features
+      - name: Run 'cargo check --no-default-features'
+        uses: actions-rs/cargo@v1
+        with:
+          command: check
+          args: --no-default-features
       - name: Run 'cross test'
         uses: actions-rs/cargo@v1
         with:
@@ -33,7 +43,7 @@ jobs:
           - riscv64gc-unknown-linux-gnu
   windows-tests:
       name: Test on Windows
-      runs-on: windows-latest 
+      runs-on: windows-latest
       steps:
         - uses: actions/checkout@v2
         - name: Install stable Rust toolchain
@@ -42,6 +52,16 @@ jobs:
             override: true
             profile: minimal
             toolchain: stable
+        - name: Run 'cargo check'
+          uses: actions-rs/cargo@v1
+          with:
+            command: check
+            args: --all-features
+        - name: Run 'cargo check --no-default-features'
+          uses: actions-rs/cargo@v1
+          with:
+            command: check
+            args: --no-default-features
         - name: Run 'cargo test'
           uses: actions-rs/cargo@v1
           with:
@@ -49,7 +69,7 @@ jobs:
             args: --all-features
   macos-tests:
       name: Test on macOS
-      runs-on: macos-latest 
+      runs-on: macos-latest
       steps:
         - uses: actions/checkout@v2
         - name: Install stable Rust toolchain
@@ -58,6 +78,16 @@ jobs:
             override: true
             profile: minimal
             toolchain: stable
+        - name: Run 'cargo check'
+          uses: actions-rs/cargo@v1
+          with:
+            command: check
+            args: --all-features
+        - name: Run 'cargo check --no-default-features'
+          uses: actions-rs/cargo@v1
+          with:
+            command: check
+            args: --no-default-features
         - name: Run 'cargo test'
           uses: actions-rs/cargo@v1
           with:

